vanilla-framework/vanilla-framework
# Remove sidebar navigation and docs example

## Done

Removed the sidebar navigation component.
Also removed the "docs" layout example since it's only purpose was showcasing this component.

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/
- See that the component no longer exists

## Details

Fixes #1623
